jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/StartCaseCommand.java

```diff
@@ -112,10 +111,7 @@ public void matchCreated(MatchCreatedEvent event) {
         });
         commands.add(commandsFactory.newInsert(caseFile));
         commands.add(commandsFactory.newFireAllRules());
-
-        BatchExecutionCommand batch = commandsFactory.newBatchExecution(commands);
-        processService.execute(deploymentId, CaseContext.get(caseId), batch);
-
+        processService.execute(deploymentId, CaseContext.get(caseId), commandsFactory.newBatchExecution(commands));
```

Comment:
unneeded

Reply:
what was unnededed is the definition of a variable which is used in one place, this is an improvement of previous code and should stay

Reply:
Change rolled back